### PR TITLE
Add Alfred 5 Early Access program

### PR DESCRIPTION
**Pull request by unitof**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference). **See [comment](https://github.com/Homebrew/homebrew-cask-versions/pull/14257#issuecomment-1176418349) below**
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by unitof** [comment]
I named this `alfred-beta` for consistency with other casks, **but** if it fits with cask-versions naming standards, I propose renaming this to `alfred-earlyaccess`, or similar.

This is not just a beta but an **Early Access* program for the forthcoming major version of this software, while it's entirely possible the current stable major version (Alfred 4) could still get its own beta releases.

Same question as I had for https://github.com/Homebrew/homebrew-cask-versions/pull/11691#issuecomment-901603890: 1Password 7 (current stable major version) continues to get both stable *and* beta releases while 1Password 8 is in an "Early Access" program.

**Comment by bevanjkay** [comment]
@unitof I agree `alfred-earlyaccess` is likely more suitable here.

**Comment by unitof** [comment]
On it!

File: Casks/alfred-earlyaccess.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+cask "alfred-earlyaccess" do
```
Comment (carlocab) [inline Casks/alfred-earlyaccess.rb:1]: I feel like we should've called this `alfred5`.

Comment (bevanjkay) [reply Casks/alfred-earlyaccess.rb:1]: @carlocab I'm happy to go with that instead. I can open the PR to get it sorted.

**Pull request merged**